app/views/facility_orders/show.html.haml

```diff
@@ -27,7 +27,7 @@
   = render "order_table", order_details: @order_details, cross_core: false, facility_id: current_facility.id
   = render "order_table_footer", order: @order, cross_core: false
 
-%h2= t("views.facility_orders.show.cross_core_orders_header")
+%h2= t("views.facility_orders.show.add_to_order_header")
```

Comment:
Could you check if the order is cross core, then select a different translation based on that?

Reply:
For example: Add to Order or Add to Cross Core Order